### ISSUE TITLE
Dynamic Tutorials

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#15217]
We should add new, dynamic tutorials for new players.

Like:
1. Open backpack for first time and get a popup explaining our inventory managment shortcuts that most people still don't know.

2. First craft: Popup that explains that items will not be consumed immediately and need to be stored near the machine to be taken in. I assume a lot of people will have the first stuff in their backpack, craft something, go away into the woods to gather something, come back and see nothing has been crafted. Possible situation for frustration -> ragequit, negative review.

**Comment by SlayksWood** [comment]
I agree the tutorials should be reworked seriously. Many features in the game that are nowhere to be seen, especially for newbies

**Comment by elfl0rd** [comment]
Please do not forget the option to turn off training at the client level.
Pop-up messages will annoy active players who see them in the tenth... the hundredth time.

**Comment by Praefator** [comment]
As I suggested such a thing a while back, could this be expanded to include Ecopedia-style customizability?

For reference:
![custom tutorial](https://user-images.githubusercontent.com/841990/77093778-cc1f2880-6a0b-11ea-9d15-a6e42fc96fcc.JPG)
